pkg/components/managementconsole/component.go

```diff
@@ -22,6 +22,49 @@ var ManagementConsoleComponent = Component{
 			{
 				IncludeAll: []string{"bz-Management Console"},
 			},
+			{Suite: "Administration pages pesudo translation"},
+			{Suite: "Branding check"},
+			{Suite: "Debug console for pods"},
+			{Suite: "Dynamic Plugins notification features"},
+			{Suite: "Dynamic plugins features"},
+			{Suite: "Dynamic provisioning"},
+			{Suite: "Home related pages via admin console"},
+			{Suite: "Notification drawer tests"},
+			{Suite: "Operators Installed page test"},
+			{Suite: "PDB List Page and Detail Page Test"},
+			{Suite: "Projects dropdown tests"},
+			{Suite: "about cluster setting page"},
+			{Suite: "add idp from console"},
+			{Suite: "admin console api related"},
+			{Suite: "alerts browser"},
```

Comment:
see from https://github.com/openshift-eng/ci-test-mapping/pull/85/files#r1607567823
`alerts browser` belongs to monitoring team